app/konnect-compatibility.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "{{site.konnect_short_name}} compatibility"
+description: 'Details which browsers, software, and versions {{site.konnect_short_name}} is compatible with.'
+content_type: reference
+layout: reference
+products:
+    - gateway
+
+tags:
+  - compatibility
+
+works_on:
+  - konnect
+
+  
+related_resources:
+  - text: "{{site.base_gateway}}"
+    url: /gateway/
+  - text: "Mesh Manager"
+    url: /mesh-manager/
+  - text: "decK"
+    url: /deck/
+  - text: "Plugins"
+    url: /plugins/
+---
+
+This reference explains which browsers, software versions, tools, and applications {{site.konnect_short_name}} is compatible with.
+
+## Supported browsers
```

Comment:
I'd move this section below all the version info + above plugin compat. It's the least  informative one.